packages/core-interfaces/src/shared/config.ts

```diff
@@ -0,0 +1,19 @@
+import get from "lodash/get";
+import set from "lodash/set";
+
+export class Config {
+    private config : {};
+
+    public init(options: any): void {
+        this.config = {};
+        Object.assign(console, options)
```

Comment:
I think I copied some specific implementation from one of the modules, but I haven't used this class in any other modules yet. It looked like a good candidate for a "shared" class because multiple modules each implement some variant of it.

Reply:
That implementation is erroneous anyway since Object.assign doesn't perform a deep-copy, if the intention was to copy stuff.